src/stream/src/executor/global_simple_agg.rs

```diff
@@ -64,6 +67,10 @@ pub struct SimpleAggExecutor<S: StateStore> {
     /// An operator will support multiple aggregation calls.
     agg_calls: Vec<AggCall>,
 
+    /// Relational state tables used by this executor.
+    /// One-to-one map with AggCall.
+    state_tables: Vec<StateTable<S>>,
```

Comment:
Can we remove the `keyspace: Vec<Keyspace<S>>`?

Reply:
After we also apply this refactor to TableValueState, yes